config-server/src/main/resources/bootstrap-local.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  cloud:
+    config:
```

Comment:
for the config server itself. isnt lines 3-4 redundant?

Reply:
Agree. Fixed